composer.json

```diff
@@ -15,9 +15,11 @@
     "wp-coding-standards/wpcs": "^2",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
     "phpunit/phpunit": "^7",
-    "phpcompatibility/php-compatibility": "^9.3"
+    "phpcompatibility/php-compatibility": "^9.3",
+    "danielbachhuber/composer-lock-updater": "^0.8.1"
```

Comment:
@danielbachhuber `clu` is installed globally in the `quay.io/pantheon-public/build-tools-ci:6.x` Dockerfile. Please use that rather than adding `clu` again as a Composer dependency

Reply:
Removed with ab2d3256ec5e85e0242ed764272a25213504d445